pkgs/top-level/all-packages.nix

```diff
@@ -8525,6 +8525,8 @@ with pkgs;
 
   rst2html5 = callPackage ../tools/text/rst2html5 { };
 
+  rstcheck = python3Packages.callPackage ../development/python-modules/rstcheck { };
```

Comment:
Add it to `python-packages.nix` and use `rstcheck = with python3Packages; toPythonApplication rstcheck;` here.

Reply:
Moved to [here](https://github.com/staccato/nixpkgs/blob/3ede328da48896f6c397ed2b21d782e6a07d1686/pkgs/top-level/python-packages.nix#L9753)